pkg/util/sys/device.go

```diff
@@ -42,6 +41,44 @@ type Partition struct {
 	Label string
 }
 
+// LocalDevice contains information about an unformatted block device
+type LocalDisk struct {
+	// Name is the device name
+	Name string `json:"name"`
+	// Parent is the device parent's name
+	Parent string `json:"parent"`
+	// HasChildren is whether the device has a children device
+	HasChildren bool `json:"hasChildren"`
+	// DevLinks is the persistent device path on the host
+	DevLinks string `json:"devLinks"`
+	// Size is the device capacity in byte
+	Size uint64 `json:"size"`
+	// UUID is used by /dev/disk/by-uuid
+	UUID string `json:"uuid"`
+	// Serial is the disk serial used by /dev/disk/by-id
+	Serial string `json:"serial"`
+	// Type is disk type
+	Type string `json:"type"`
+	// Rotational is the boolean whether the device is rotational: true for hdd, false for ssd and nvme
+	Rotational bool `json:"rotational"`
+	// ReadOnly is the boolean whether the device is readonly
+	Readonly bool `json:"readOnly"`
+	// OwnPartition is whether rook owns the partition
+	OwnPartition bool `json:"ownPartition"`
+	// Filesystem is the filesystem currently on the device
+	Filesystem string `json:"filesystem"`
```

Comment:
the `filesystem` property was blank in the configmap even for my root drive

Reply:
if the root device has partitions then there is no filesystem